tests/test_connections/test_oef/test_communication.py

```diff
@@ -319,72 +319,89 @@ def test_cfp(self):
         """Test that a CFP can be sent correctly."""
         cfp_bytes = FIPAMessage(message_id=0, dialogue_reference=(str(0), ''), target=0, performative=FIPAMessage.Performative.CFP,
                                 query=Query([Constraint('something', ConstraintType('>', 1))]))
+        cfp_bytes.counterparty = self.crypto1.public_key
```

Comment:
the counterparty is crypto2 ...it doesn't really matter but we should avoid being inconsistent